api/src/main/java/io/opentelemetry/api/trace/StatusCode.java

```diff
@@ -5,36 +5,18 @@
 
 package io.opentelemetry.api.trace;
 
-/**
- * The set of canonical status codes. If new codes are added over time they must choose a numerical
- * value that does not collide with any previously used value.
- */
+/** The set of canonical status codes. */
 public enum StatusCode {
 
   /**
    * The operation has been validated by an Application developers or Operator to have completed
    * successfully.
    */
-  OK(0),
+  OK,
 
   /** The default status. */
-  UNSET(1),
+  UNSET,
 
   /** The operation contains an error. */
-  ERROR(2);
-
-  private final int value;
-
-  StatusCode(int value) {
-    this.value = value;
-  }
-
-  /**
-   * Returns the numerical value of the code.
-   *
-   * @return the numerical value of the code.
-   */
-  public int value() {
```

Comment:
since this is a public API should we deprecate it before deleting?

Reply:
Makes sense